pattoo/api/portal/users.py

```diff
@@ -0,0 +1,89 @@
+"""Routes for users."""
+
+# Flask imports
+from flask import Blueprint
+from flask import request, session
+
+# Pattoo imports
+from pattoo.db import db
+from pattoo.db.models import User as UserModel
+
+# Import server resources
+# from .forms import AddUserForm
+
+# Define the AGENTS global variable
+USERS = Blueprint('USERS', __name__)
+
+@USERS.route('/api/user', methods=['GET'])
+def user():
+    """User route.
+    
+    Args:
+        None
+
+    Returns:
+        response (dict): Response Message
+
+    """
+    # Check if a user was stored in session first
+    if session.get('idx_user', None) is None:
+        response = {'data': {'message': 'Login first'}}
+        # Block access if no user was found in session
+        return response
+
+    enable = request.args.get('enable', type=int)
+    user_id = request.args.get('user', type=int)
+    if enable is not None and user_id is not None:
+        # Toggle enable value of enable sent from request
+        change_enable = 1 if (enable == 0) else 0
+        response = {'data': {'user_id': user_id, 'enable': change_enable, 'message': 'Not changed'}}
+        with db.db_modify(20185, die=True) as db_session:
+            # Change enable status of user
+            db_session.query(UserModel).filter(
+                UserModel.idx_user == user_id
+            ).update({'enabled': change_enable})
+            response = {'data': {'user_id': user_id, 'enable': change_enable, 'message': 'Changed'}}
+        return response
+    
+    response = {'data':{'message': 'Query did not run'}}
```

Comment:
In general initialize the variable at the top of the function. If it changes, return that new value immediately. It makes troubleshooting and code flow much easier.